modules/nodes-cluster-resource-configure-jdk.adoc

```diff
@@ -53,65 +53,64 @@ This sets `-XX:MaxRAM` to the container memory limit, and the maximum heap size
+// workloads, but notable exceptions include workloads where additional active
+// processes co-exist with a JVM within a container, whether those additional
+// processes are native, additional JVMs, or a combination of the two.
+//
+// The {product-title} Jenkins maven slave image uses the following JVM arguments to encourage the JVM
+// to release unused memory to the operating system:
+//
+// [source,terminal]
+// ----
+// -XX:+UseParallelGC
+// -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4
+// -XX:AdaptiveSizePolicyWeight=90.
+// ----
+//
+// These arguments are intended to return heap
+// memory to the operating system whenever allocated memory exceeds 110% of in-use
+// memory (`-XX:MaxHeapFreeRatio`), spending up to 20% of CPU time in the garbage
+// collector (`-XX:GCTimeRatio`). At no time will the application heap allocation
+// be less than the initial heap allocation (overridden by `-XX:InitialHeapSize` /
+// `-Xms`). Detailed additional information is available
+// link:https://developers.redhat.com/blog/2014/07/15/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-1/[Tuning Java's footprint in OpenShift (Part 1)],
+// link:https://developers.redhat.com/blog/2014/07/22/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-2/[Tuning Java's footprint in OpenShift (Part 2)],
+// and at
+// link:https://developers.redhat.com/blog/2017/04/04/openjdk-and-containers/[OpenJDK
+// and Containers].
+
+// [id="nodes-cluster-resource-configure-jdk-proc_{context}"]
+// == Understanding how to ensure all JVM processes within a container are appropriately configured
+//
+// In the case that multiple JVMs run in the same container, it is essential to
+// ensure that they are all configured appropriately. For many workloads it will
+// be necessary to grant each JVM a percentage memory budget, leaving a perhaps
+// substantial additional safety margin.
+//
+// Many Java tools use different environment variables (`JAVA_OPTS`, `GRADLE_OPTS`, and so on) to configure their JVMs and it can be challenging to ensure
+// that the right settings are being passed to the right JVM.
+//
+// The `JAVA_TOOL_OPTIONS` environment variable is always respected by the OpenJDK,
+// and values specified in `JAVA_TOOL_OPTIONS` will be overridden by other options
+// specified on the JVM command line. By default, to ensure that these options are
+// used by default for all JVM workloads run in the slave image, the {product-title} Jenkins
+// maven slave image sets:
+//
+// [source,terminal]
+// ----
+// JAVA_TOOL_OPTIONS="-XX:+UnlockExperimentalVMOptions
+// -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true"
+// ----
+//
+// [NOTE]
+// ====
+// The `UseCGroupMemoryLimitForHeap` option has been removed in JDK 11. Use `-XX:+UseContainerSupport` instead.
+// ====
+//
+// This does not guarantee that additional options are not required, but is
+// intended to be a helpful starting point.
```

Comment:
Valid if rephrased correctly (see previously)

Reply:
I think I got this right. Please take another look at the result.

Reply:
that all looks good now @rolfedh 
I will lgtm